docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -512,6 +519,29 @@ The Iceberg JDBC catalog is supported for the Iceberg connector.  At a minimum,
 database besides PostgreSQL, a JDBC driver jar file must be placed in the plugin
 directory.
 
+:::{list-table} JDBC catalog configuration properties
+:widths: 40, 60
+:header-rows: 1
+
+* - Property name
+  - Description
+* - `iceberg.jdbc-catalog.driver-class`
+  - JDBC driver class name.
+* - `iceberg.jdbc-catalog.connection-url`
+  - The URI to connect to the JDBC server.
```

Comment:
Whats the syntax to use here. Add an example .. assume the reader does not what a JDBC connection URL is